frontend/src/i18n/translations/en-US.ts

```diff
@@ -199,6 +199,14 @@ export default {
   "account.reverseShares.modal.send-email.description":
     "Send an email notification when a share is created with this reverse share link.",
 
+  "account.reverseShares.modal.simplified": "Simple mode",
+  "account.reverseShares.modal.simplified.description":
+    "Make it easy for the person uploading the file to share it with you. They will not be able to customize the share link and additional security options.",
+
+  "account.reverseShares.modal.public-access": "Public access",
+  "account.reverseShares.modal.public-access.description":
+    "Anyone who knows the share link can access this shared file. When this switch is on, the file uploader can share their link with anyone.",
```

Comment:
When disabled, both `share.creator` and `reverseShare.creator` can be accessed. If `share.creator` is not reflected in the description, will it cause misunderstanding?

@stonith404

Reply:
However, the creator may be anonymous, and the description may become complex if it is to fully reflect the actual situation.

Reply:
Yeah good point. What do you think about
> Make the created shares with this reverse share public. If disabled only you and the signed in creator can view the created shares.

?

Reply:
In fact, I don't know how to describe it, but since this description is just a hint to the creator of the reverse share, maybe it should simply tell him something like "only you and the creator of the share can view it". Express only one risk, rather than a comprehensive description. 

Do you have any ideas? Or leave it as it is and wait for someone to raise an issue?